_includes/patterns-filter.html

```diff
@@ -331,7 +331,7 @@
     let whiteImg = data[i].white;
 
     label.innerHTML = `<img src="${colorImg}" id="logo-dark-light" data-logo-for-dark="${whiteImg ?? colorImg}"
-                    data-logo-for-light="${colorImg}" /><div class="name">${data[i].name}</div><div class="badge">${count}</div>`;
+                    data-logo-for-light="${colorImg}" /><div class="name">${data[i].name.replace(/-/g, " ")}</div><div class="badge">${count}</div>`;
```

Comment:
@iArchitSharma We don't need string manipulation, is name only property we get in data? do we get displayName? I will say we have can get displayName please use the instead doing string manipulation with name

Reply:
@Yashsharma1911 we don't have any displayName or similar property
![image](https://github.com/meshery/meshery.io/assets/74408634/0de10a73-d963-4323-b6bd-fba39b22c0d7)